### PR TITLE
util-linux

### PR DESCRIPTION
**Pull request by scopatz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scopatz** [comment]
This is a linux only recipe, that is ready for review @conda-forge/staged-recipes!

File: recipes/util-linux/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            --disable-nologin    \
+            --disable-su         \
+            --disable-setpriv    \
+            --disable-runuser    \
+            --disable-static     \
+            --without-systemd    \
+            --disable-makeinstall-chown \
+            --disable-makeinstall-setuid \
+            --without-systemdsystemunitdir
+make -j ${CPU_COUNT}
```
Comment (isuruf) [inline recipes/util-linux/build.sh]: Run `make check`

File: recipes/util-linux/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - ncurses
+    - readline
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libmount.so"
+
+about:
+  home: https://en.wikipedia.org/wiki/Util-linux
+  license: GPL v2.0
```
Comment (chrisburr) [inline recipes/util-linux/meta.yaml]: https://github.com/karelzak/util-linux/blob/master/libmount/COPYING
```suggestion
  license: LGPL-2.0-or-later
```

**Comment by scopatz** [comment]
Ready for review again @conda-forge/staged-recipes!

**Pull request merged**